src/steps/http.ts

```diff
@@ -286,10 +291,6 @@ export default async function (
       const appendOptions = {} as FormData.AppendOptions
       if (typeof params.formData[field] === 'string') {
```

Comment:
One thing: Can we also check for boolean and number? They should be converted automatically to string if you pass them to formData.append

Reply:
done, please check